Input/source/input.css

```diff
@@ -0,0 +1,110 @@
+@import url("../../global.css");
+
+.form {
+  background-color: var(--primary-text-color);
+  width: 30rem;
+  padding: 2em;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 2em;
+  box-shadow: var(--universal-box-shadow);
+}
+.form-field {
+  width: 100%;
+}
+
+.input-lbl {
+  display: block;
+  margin-bottom: 0.2em;
+}
+
+.required-field::after {
+  content: "*";
+  color: var(--cta-background-color);
+}
+
+.form-input {
+  all: unset;
+  border-radius: var(--border-radius-custom);
+  background-color: var(--grey-text-color);
+  padding: 0.5em;
+  border-image: none;
+  border-color: transparent;
+  border-width: 0;
+  border-style: none;
+  outline: 3px solid var(--light-text-color);
+  color: inherit;
+  font-style: inherit;
+
+  width: 100%;
+}
+
+.form-input:focus {
+  outline-color: var(--cta-background-color);
+  box-shadow: var(--universal-box-shadow);
+}
+.form-textarea {
+  all: unset;
+  border-radius: var(--border-radius-custom);
+  background-color: var(--light-text-color);
+  outline: 3px solid var(--light-text-color);
+  color: inherit;
+  font-style: inherit;
+  display: block;
+  padding: 0.5em;
+
+  overflow: auto;
+}
+
+.form-textarea:focus {
+  outline-color: var(--cta-background-color);
+  box-shadow: var(--universal-box-shadow);
+}
+
+#form-submit:disabled {
+  cursor: not-allowed;
+  background-color: var(--danger-text-color);
+}
+
+.txtarea-mdm {
+  height: 10em;
+  width: 20em;
+}
+
+.txtarea-sml {
+  height: 5em;
+  width: 10em;
+}
+
+.txtarea-lrg {
+  height: 20em;
+  width: 40em;
+}
+
+.padding-sml {
+  padding: 0.5em;
+}
+
+.display-none {
+  display: none;
+}
+
+.display-block {
+  display: block;
+}
```

Comment:
`display-none` & `display-block` can be used in many places consider shifting these classes to the global stylesheet in the future maybe.